packages/app/src/vis-packs/core/complex/ComplexLineToolbar.tsx

```diff
@@ -43,11 +44,13 @@ function ComplexLineToolbar(props: Props) {
         label="X"
         value={xScaleType}
         onScaleChange={setXScaleType}
+        options={AXIS_SCALE_TYPES}
```

Comment:
Note that we had forgotten to pass `options` arrays, which meant that all scales were shown in the drop-down. Since `xScaleType` is now typed to `AxisScaleType`, we get proper errors when we don't pass `options`.